src/services/jobs/dto/CreateJob.input.ts

```diff
@@ -0,0 +1,31 @@
+import { Field, Float, InputType } from '@nestjs/graphql';
+
+@InputType()
+export class CreateJobInput {
+  @Field()
+  user_client_id: string;
+
+  @Field({ nullable: true })
+  user_worker_id: string;
+
+  @Field()
+  job_type: string;
+
+  @Field()
+  status: string;
+
+  @Field()
+  description: string;
+
+  @Field()
+  created_at: Date;
```

Comment:
you don't need to request the created at attribute from the client, this should be generated from the server

Reply:
You are right, i will fix it